resources/building-mobile-extensions/event-listeners.md

```diff
@@ -1,7 +1,10 @@
 # Listening for events
 
+
 A common use case is to add an event listener to get notifications for events. The main location to add an event listener is in the `init` method, although you can add listeners by using other callbacks later. You can add the logic that you want executed when an event occurs, and for which you have a listener, in the `hear` method of your listener class.
 
+In the Adobe Experience Cloud Platform SDK, the strings  like extension name, event type and source strings are converted and stored in lowercase. When you compare strings use java.lang.String.equalsIgnoreCase() method.
```

Comment:
Let's move this after the next paragraph and reword it to something like:
`The event type and event source associated with the received event may be lower-cased as the Mobile SDK compares them as case insensitive values. It is recommended to use ignore case string comparison when checking the event type and event source of the event.`

Reply:
@emdobrin and @samthiru, there is no term such as "lower cased", and the 2nd sentence uses passive text. I'd like to reword it to:

"The event type and event source that are associated with the received event can be in lower case because the Mobile SDK compares the type and the source as case insensitive values. When you check the event type and event source of the event, we recommend that you use the ignore case string comparison."